### ISSUE TITLE
CMake fails to build (Windows 10)

### ISSUE THREAD

**Issue by VisComKreiser** [inviwo/inviwo#65]
Hi,
I tried to build the project but a CMake error occurs while building for Visual Studio 2015.
Im using CMake 3.7.0 on my desktop machine and 3.11.0 on my laptop and both times the same error:

```
CMake Error at D:/Repositories/inviwo-github-release/build/CMakeFiles/CMakeTmp/CMakeLists.txt:14 (add_library):
  Cannot find source file:

    /cmake/findheader.cpp

  Tried extensions .c .C .c++ .cc .cpp .cxx .m .M .mm .h .hh .h++ .hm .hpp
  .hxx .in .txx

CMake Error: CMake can not determine linker language for target: cmTC_14424
CMake Error: Internal CMake error, TryCompile generation of cmake failed
CMake Error at cmake/globalmacros.cmake:626 (message):
  Header path not found
Call Stack (most recent call first):
  cmake/globalmacros.cmake:660 (ivw_get_header_path)
  cmake/globalconfig.cmake:107 (include)
  CMakeLists.txt:90 (include)
```

**Comment by petersteneteg** [comment]
Question: do you have the build folder inside the source folder? We generally build “out of source” and I suspect we might have issues with in source builds. Should probably add a check for that in cmake

**Comment by VisComKreiser** [comment]
Yes, I'm having my build directory inside the source folder, i'll try to build it out of source then

**Comment by VisComKreiser** [comment]
there is still the same issue when i have my sources in `D:/Repositories/inviwo-github-release` and the build in `D:/builds/inviwo-github-release-build`
I also deleted the build folder in my source directory

**Comment by petersteneteg** [comment]
Does D:/builds/inviwo-github-release-build/cmake/findheader.cpp exist?

**Comment by VisComKreiser** [comment]
no, CMake produces the error quite early when i try to configure.
There is only a CMakeFiles directory with some stuff in it and CMakeCache.txt file.

**Comment by petersteneteg** [comment]
Found the issue, IVW_BINARY_DIR was read before it was set. Should be fixed as soon as PR #68 is merged.
Later today I hope.

**Comment by VisComKreiser** [comment]
thanks for your help
